### PR TITLE
alpine: bump 3.4.6, add 3.5.0 and update edge

### PR DESCRIPTION
**Pull request by ncopa**

### CODE REVIEW

File: library/alpine [hunk 1]
```diff
@@ -1,9 +1,10 @@
...
-3.2: git://github.com/gliderlabs/docker-alpine@5636e9a91b543fa507271fbbc9b5d1db59ce986f versions/library-3.2
-3.3: git://github.com/gliderlabs/docker-alpine@bc90ea0c90880b44cc7f405cbe6ec27ed453fda7 versions/library-3.3
-3.4: git://github.com/gliderlabs/docker-alpine@a83f6c797a617f580ff717efe9e53543f7beca43 versions/library-3.4
-latest: git://github.com/gliderlabs/docker-alpine@a83f6c797a617f580ff717efe9e53543f7beca43 versions/library-3.4
-edge: git://github.com/gliderlabs/docker-alpine@64804257e6997cdc88c39e949b6a9df492621691 versions/library-edge
+3.1: git://github.com/gliderlabs/docker-alpine@6a7732123b1e22c836e7ff538828ecf9eb0ebbb3 versions/library-3.1
+3.2: git://github.com/gliderlabs/docker-alpine@39157ab3587065a0e0928663dea293ae43110cd9 versions/library-3.2
+3.3: git://github.com/gliderlabs/docker-alpine@62168e0f780d64f2124a7be94ed31991a31ae5ad versions/library-3.3
+3.4: git://github.com/gliderlabs/docker-alpine@71c9ce882016759bcc0d47f372d20a1b6ec93cd5 versions/library-3.4
+latest: git://github.com/gliderlabs/docker-alpine@71c9ce882016759bcc0d47f372d20a1b6ec93cd5 versions/library-3.4
```
Comment (rochacon) [inline library/alpine]: Should this be updated to 3.5 also?

Comment (ncopa) [inline library/alpine]: yes, it should. Just wanted push the first image and double check that it works before making a change that affects alot of users.

**Comment by tianon** [comment]
```diff
warning: insecure protocol git:// detected: git://github.com/gliderlabs/docker-alpine
diff --git a/alpine_3.1/rootfs.tar.gz b/alpine_3.1/rootfs.tar.gz
index 9d42ff5..4aa1a55 100644
Binary files a/alpine_3.1/rootfs.tar.gz and b/alpine_3.1/rootfs.tar.gz differ
diff --git a/alpine_3.2/rootfs.tar.gz b/alpine_3.2/rootfs.tar.gz
index cd5306c..0b3c89c 100644
Binary files a/alpine_3.2/rootfs.tar.gz and b/alpine_3.2/rootfs.tar.gz differ
diff --git a/alpine_3.3/rootfs.tar.gz b/alpine_3.3/rootfs.tar.gz
index 4a94217..71dff7b 100644
Binary files a/alpine_3.3/rootfs.tar.gz and b/alpine_3.3/rootfs.tar.gz differ
diff --git a/alpine_latest/Dockerfile b/alpine_3.5/Dockerfile
similarity index 100%
copy from alpine_latest/Dockerfile
copy to alpine_3.5/Dockerfile
diff --git a/alpine_edge/rootfs.tar.gz b/alpine_3.5/rootfs.tar.gz
similarity index 21%
copy from alpine_edge/rootfs.tar.gz
copy to alpine_3.5/rootfs.tar.gz
index 464dcb5..fe12bab 100644
Binary files a/alpine_edge/rootfs.tar.gz and b/alpine_3.5/rootfs.tar.gz differ
diff --git a/alpine_edge/rootfs.tar.gz  'tar -t' b/alpine_3.5/rootfs.tar.gz  'tar -t'
similarity index 99%
copy from alpine_edge/rootfs.tar.gz  'tar -t'
copy to alpine_3.5/rootfs.tar.gz  'tar -t'
index a196a43..50b455b 100644
--- a/alpine_edge/rootfs.tar.gz  'tar -t'	
+++ b/alpine_3.5/rootfs.tar.gz  'tar -t'	
@@ -93,7 +93,7 @@
 ./etc/apk/keys/alpine-devel@lists.alpinelinux.org-5243ef4b.rsa.pub
 ./etc/apk/keys/alpine-devel@lists.alpinelinux.org-524d27bb.rsa.pub
 ./etc/apk/keys/alpine-devel@lists.alpinelinux.org-5261cecb.rsa.pub
-./etc/apk/keys/alpine-devel@lists.alpinelinux.org-57a8ad0e.rsa.pub
+./etc/apk/keys/alpine-devel@lists.alpinelinux.org-58199dcc.rsa.pub
 ./etc/apk/protected_paths.d/
 ./etc/apk/repositories
 ./etc/apk/world
diff --git a/alpine_edge/rootfs.tar.gz b/alpine_edge/rootfs.tar.gz
index 464dcb5..d0afdac 100644
Binary files a/alpine_edge/rootfs.tar.gz and b/alpine_edge/rootfs.tar.gz differ
diff --git a/alpine_edge/rootfs.tar.gz  'tar -t' b/alpine_edge/rootfs.tar.gz  'tar -t'
index a196a43..1c983e1 100644
--- a/alpine_edge/rootfs.tar.gz  'tar -t'	
+++ b/alpine_edge/rootfs.tar.gz  'tar -t'	
@@ -93,7 +93,7 @@
 ./etc/apk/keys/alpine-devel@lists.alpinelinux.org-5243ef4b.rsa.pub
 ./etc/apk/keys/alpine-devel@lists.alpinelinux.org-524d27bb.rsa.pub
 ./etc/apk/keys/alpine-devel@lists.alpinelinux.org-5261cecb.rsa.pub
-./etc/apk/keys/alpine-devel@lists.alpinelinux.org-57a8ad0e.rsa.pub
+./etc/apk/keys/alpine-devel@lists.alpinelinux.org-58199dcc.rsa.pub
 ./etc/apk/protected_paths.d/
 ./etc/apk/repositories
 ./etc/apk/world
@@ -126,6 +126,7 @@
 ./etc/network/if-pre-down.d/
 ./etc/network/if-pre-up.d/
 ./etc/network/if-up.d/
+./etc/network/if-up.d/dad
 ./etc/opt/
 ./etc/os-release
 ./etc/passwd
diff --git a/alpine_latest/rootfs.tar.gz b/alpine_latest/rootfs.tar.gz
index 1279a34..90de5b1 100644
Binary files a/alpine_latest/rootfs.tar.gz and b/alpine_latest/rootfs.tar.gz differ
```

**Comment by tianon** [comment]
cc @caervs @toli

**Comment by tianon** [comment]
LGTM

Build test of #2489; 98c9644 (`alpine`):

```console

$ bashbrew build alpine:3.1
warning: insecure protocol git:// detected: git://github.com/gliderlabs/docker-alpine
Building bashbrew/cache:50e7453d496f3ff399035afb907d2bc64d72a1c8e1e56b70f5aef755bab107ec (alpine:3.1)
Tagging alpine:3.1

$ test/run.sh alpine:3.1
testing alpine:3.1
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build alpine:3.2
Building bashbrew/cache:0b5ab413d265be7a427d025cdc6315f18d147a6d4e8b140a8c000c85723b1e00 (alpine:3.2)
Tagging alpine:3.2

$ test/run.sh alpine:3.2
testing alpine:3.2
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build alpine:3.3
Building bashbrew/cache:900e821ab4ec596c9651010b0cf89f753e00907bd3f909381161e7017ea66bb3 (alpine:3.3)
Tagging alpine:3.3

$ test/run.sh alpine:3.3
testing alpine:3.3
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build alpine:3.4
Building bashbrew/cache:8321b2f6286ae4084424dda9908b80302c6f411779354e1c88472a62968ceecf (alpine:3.4)
Tagging alpine:3.4
Tagging alpine:latest

$ test/run.sh alpine:3.4
testing alpine:3.4
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build alpine:3.5
Building bashbrew/cache:a20ba743a1ae51ab036fe1035df9a11ec92776efff7a142462ad874ea310ada2 (alpine:3.5)
Tagging alpine:3.5

$ test/run.sh alpine:3.5
testing alpine:3.5
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build alpine:edge
Building bashbrew/cache:a211f6675ee97d7d4e4de6aab6b355f3c6a108480db423aba15d70f81c0f4980 (alpine:edge)
Tagging alpine:edge

$ test/run.sh alpine:edge
testing alpine:edge
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed

```

**Pull request merged**